front/app/containers/Admin/projects/project/offlineInputs/index.tsx

```diff
@@ -97,11 +98,11 @@ const OfflineInputImporter = () => {
       ...supportedFormData
     } = formData;
 
-    const location_point_geojson =
-      typeof location_description === 'string' &&
-      location_description.length > 0
-        ? await geocode(location_description)
-        : undefined;
+    // const location_point_geojson =
+    //   typeof location_description === 'string' &&
+    //   location_description.length > 0
+    //     ? await geocode(location_description)
+    //     : undefined;
```

Comment:
Just making sure we don't forget the un-comment these

Reply:
Yeah I forgot to say that I had to comment those out because it was preventing me from approving. Is that because I don't have the geocoder enabled? Also, the xlsx import allows you to define the long/lat values, so this code should be ignored I think if they are already present